tests/Unit/Filament/Resources/Locations/LocationResourceTest.php

```
@@ -0,0 +1,67 @@
+
+use App\Filament\Resources\Locations\LocationResource;
+use App\Models\Location;
+use Filament\Support\Icons\Heroicon;
+use PHPUnit\Framework\TestCase;
+
+class LocationResourceTest extends TestCase
+{
+    #[\PHPUnit\Framework\Attributes\Test]
+    public function should_return_correct_model_class(): void
+    {
+        $resource = new LocationResource;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$resource'. (undefined)

(UnusedLocalVariable)